tt_metal/hw/inc/grayskull/noc_nonblocking_api.h

```diff
@@ -19,42 +19,21 @@ extern uint32_t noc_reads_num_issued[NUM_NOCS];
 extern uint32_t noc_nonposted_writes_num_issued[NUM_NOCS];
 extern uint32_t noc_nonposted_writes_acked[NUM_NOCS];
 
-inline void NOC_CMD_BUF_WRITE_REG(uint32_t noc, uint32_t buf, uint32_t addr, uint32_t val) {
+inline __attribute__((always_inline)) void NOC_CMD_BUF_WRITE_REG(uint32_t noc, uint32_t buf, uint32_t addr, uint32_t val) {
```

Comment:
can we keep consistent and use FORCE_INLINE?

Reply:
FORCE_INLINE is defined in dataflow_api.h (it shouldn't be) which noc_nonblockinga_api.h is a child of so can't use.  I'd have to redefine it or move the ifdef.  best fix is to create a header at a lower level and move it there.
(btw: dataflow_api.h defines a fn "min" too, ugh)